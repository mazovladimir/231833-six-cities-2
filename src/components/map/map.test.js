import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

const offers = [{image: `image`, priceValue: 100, priceText: `text`, name: `name`, type: `type`, coordinates: [1, 2]}];

it(`renders correct map component`, () => {
  const tree = renderer.create(<Map offers={offers} />).toJSON();
  expect(tree).toMatchSnapshot();
});
