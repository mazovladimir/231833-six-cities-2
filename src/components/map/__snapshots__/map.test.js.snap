// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correct map component 1`] = `
<div
  id="map"
/>
`;
